Sources/SmileID/Classes/Networking/Models/UploadRequest.swift

```diff
@@ -2,9 +2,35 @@ import Foundation
 
 public struct UploadRequest: Codable {
     var images: [UploadImageInfo]
+    var idInfo: IdInfo?
 
     enum CodingKeys: String, CodingKey {
         case images
+        case idInfo = "id_info"
+    }
+}
+
+public struct IdInfo: Codable {
+    var country: String
+    var idType: String
+    var idNumber: String?
+    var firstName: String?
+    var middleName: String?
+    var lastName: String?
+    var dob: String?
+    var bankCode: String?
+    var entered: Bool?
+
+    enum  CodingKeys: String, CodingKey {
```

Comment:
nit: spacing. Can the linter catch this?
```suggestion
    enum CodingKeys: String, CodingKey {
```